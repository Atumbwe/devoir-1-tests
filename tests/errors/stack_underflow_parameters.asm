push 3
add
sub
mul
div
mod
pop
print
stack
