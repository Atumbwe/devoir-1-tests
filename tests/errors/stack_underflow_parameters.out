ERROR (add): stack underflow
ERROR (sub): stack underflow
ERROR (mul): stack underflow
ERROR (div): stack underflow
ERROR (mod): stack underflow
ERROR (print): stack underflow
[ ]
